checkov/terraform/module_loading/module_finder.py

```diff
@@ -36,6 +37,22 @@ def __str__(self) -> str:
 def find_modules(path: str) -> List[ModuleDownload]:
     modules_found: list[ModuleDownload] = []
 
+    # Leverage modules.json to better inform discovery. If we have this,
+    # there should be no need to walk and gather modules
+    if env_vars_config.CHECKOV_EXPERIMENTAL_TERRAFORM_MANAGED_MODULES:
+        tf_modules_file = Path(path) / '.terraform' / 'modules' / 'modules.json'
+        if tf_modules_file.exists():
+            for mod in json.loads(tf_modules_file.read_bytes())['Modules']:
+                if len(mod['Key']):
```

Comment:
```suggestion
                if mod['Key']:
```
it is faster to check for truthiness, which means it is not empty.